src/modules/util.js

```diff
@@ -0,0 +1,33 @@
+export default class Utilities {
+    static baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+
+    static showBaseUrl ='https://api.tvmaze.com/shows/'
+
+    static exception = (message) => message
+
+    static getHeader() {
+      const headers = new Headers();
+      headers.append('Content-Type', 'application/x-www-form-urlencoded');
+      return headers;
+    }
+
+       static getParams = (id, username, comment) => {
+         const urlencoded = new URLSearchParams();
+         urlencoded.append('item_id', `${id}`);
+         urlencoded.append('username', `${username}`);
+         urlencoded.append('comment', `${comment}`);
+         return urlencoded;
+       }
+
+        static cleanFormInput = () => {
+          document.getElementById('name').value = '';
+          document.getElementById('insights').value = '';
+        }
+
+        static showCommentCounter = (commentList) => {
+          if (commentList.length === 0) {
+            return 0;
+          }
+          return commentList.length;
+        }
+}
```

Comment:
According to the requirements, the counter should be implemented by counting the actual number of items present in the DOM. To achieve this, you can use the `querySelectorAll` method which would return an array of all specified items.Kindly take this into consideration as it is a core requirement for this milestone